go.mod

```diff
@@ -124,4 +124,4 @@ require (
 	gotest.tools/v3 v3.0.3 // indirect
 )
 
-replace github.com/docker/docker => github.com/docker/docker v20.10.3-0.20220121014307-40bb9831756f+incompatible
+replace github.com/docker/docker => github.com/docker/docker v20.10.3-0.20220224222438-c78f6963a1c0+incompatible
```

Comment:
not for this PR, but we should try again what happens if we remove the `replace` (does it revert because `master` is interpreted as `20.10.3`, which is older than `20.10.12` ? or does it "stick"?)